### ISSUE TITLE
Ability to clone or export to STL negative or positive parts of the assembly

### ISSUE THREAD

**Issue by mkotek** [SoftFever/OrcaSlicer#7421]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

It would be nice to be able to clone the negative part that is added to an assembly.

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

When quickly modyfing an existing object, like adding homes, it would be great to be able to clone the part of the assembly just like you can clone a separate part. An added benefit would be to be able to export such part to an STL, without breaking assembly to parts.

### Describe alternatives you've considered

1. Perform an action in a CAD package.
2. Split assembly to parts, clone or export a needed part and assemble parts again.

**Comment by jbss71** [comment]
yes, magnet pockets, screw holes the stuff we add to things alot we can save and load in.  im not a pro, so im sure its there. but how to use?
